servers/src/main/java/tachyon/worker/block/BlockDataManager.java

```diff
@@ -76,6 +77,10 @@ public void setHeartbeatReporter(BlockHeartbeatReporter reporter) {
     public void setMetricsReporter(BlockMetricsReporter reporter) {
       mMetricsReporter = reporter;
     }
+
+    public void setSessions(Sessions sessions) {
```

Comment:
Is this function still used in non-test code? If not, it should not be in the public API.